examples/website/index.html

```diff
@@ -4,6 +4,6 @@
   </head>
   <body>
     <h1>{{.WelcomeText}}</h1>
-    {{if .ShowLogo}}<img src="logo.png">{{end}}
+    {{if eq .ShowLogo "true"}}<img src="logo.png">{{end}}
```

Comment:
This is a quick bug fix not related to this PR, but it annoyed me enough that I had to correct it.